javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,27 @@
+let peterHouseVolume = 8 * 10 * 10; //8m wide, 10m deep, 10m height
```

Comment:
- It is nice you added comments to explain what is these numbers. 
You can also make them variables like
```js
const wide = 8;
const deep = 10;
... 
```
then calculate `peterHouseVolume`
- use `const` as long as the value will not be changed later